config/initializers/customize_error.rb

```diff
@@ -0,0 +1,12 @@
+# typed: false
+# frozen_string_literal: true
+
+ActionView::Base.field_error_proc = proc do |html_tag, _instance|
```

Comment:
Can you drop a comment here-ish about what this is used for/why we need it?

Reply:
added

Reply:
thank you